ready_recipes/geos/meta.yaml

```diff
@@ -16,7 +16,16 @@ requirements:
         - msinttypes  # [win]
     run:
 
+test:
+    commands:
+        - "[ $($PREFIX/bin/geos-config --version) == '3.4.2' ]"  # [not win]
```

Comment:
Oh, that's annoying. New-ish conda-builds don't like this line, but my conda-build did...